bucket/losslesscut.json

```
@@ -1,38 +1,32 @@
 {
     "homepage": "https://github.com/mifi/lossless-cut",
-    "description": "Save space by quickly and losslessly trimming video and audio files.",
-    "version": "2.4.0",
+    "description": "Lossless trimming tool for video and audio files.",
+    "version": "2.6.2",
     "license": "MIT",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/mifi/lossless-cut/releases/download/v2.4.0/LosslessCut-win32-x64.zip",
-            "hash": "49cf1e1a250fab8dfd42f33ce4e7d4e1de19ebd29caa975ce579cf22e7dc130f",
-            "extract_dir": "LosslessCut-win32-x64"
-        },
-        "32bit": {
-            "url": "https://github.com/mifi/lossless-cut/releases/download/v2.4.0/LosslessCut-win32-ia32.zip",
-            "hash": "0f87bd29d44266f65f30ece2af766347b4d4e16fd9cd753f24bc75ee35252f0b",
-            "extract_dir": "LosslessCut-win32-ia32"
+            "url": "https://github.com/mifi/lossless-cut/releases/download/v2.6.2/LosslessCut-2.6.2.exe#/dl.7z",
+            "hash": "2e99be1fe8be6dd6e6a10e7103635cdcd5d52dac1329c3175c68c8026e4048fc",
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\" | Out-Null",
```

There is no need for out-null
```suggestion
                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
```